composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "friendsoftypo3/headless",
+  "type": "typo3-cms-extension",
+  "version": "1.0.0-beta",
```

Comment:
remove version form here, we will tag the release on github and register it in packagist

Reply:
done